### PR TITLE
Add info about MongoDB not supporting vboxsf

### PR DESCRIPTION
**Pull request by yosifkit**
closes https://github.com/docker-library/mongo/issues/74
closes https://github.com/docker-library/mongo/issues/68

### CODE REVIEW

File: mongo/content.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Important note: There are several ways to store data used by applications that r
...
 
-**WARNING**: because MongoDB uses memory mapped files it is not possible to use it through vboxsf to your host ([vbox bug](https://www.virtualbox.org/ticket/819)).
+**WARNING**: because MongoDB uses memory mapped files it is not possible to use it through vboxsf to your host ([vbox bug](https://www.virtualbox.org/ticket/819)). VirtualBox shared folders is not supported by mongodb (see [docs.mongodb.org](https://docs.mongodb.org/manual/administration/production-notes/#fsync-on-directories) and related [jira.mongodb.org](https://jira.mongodb.org/browse/SERVER-8600) bug). This means that it is not possible with the default setup using Docker Toolbox to run a MongoDB container with the data directory mapped to the host.
```
Comment (tianon) [inline mongo/content.md]: "VirtualBox shared folders is not" -> "VirtualBox shared folders are not"
"mongodb" -> "MongoDB"

**Comment by yosifkit** [comment]
@tianon, updated!

**Comment by tianon** [comment]
LGTM :+1:

**Comment by tianon** [comment]
image

**Comment by devel0** [comment]
a workaround is to mount using bind, so for example /home/user/vboxdrv is the folder where mongodb can't write, then use

mount --bind /home/user/vboxdrv /home/user/vboxdrv2

now trying to run docker mongo with that vboxdrv2 folder works

**Pull request merged**